packages/blitz-auth/src/server/passport-adapter.ts

```diff
@@ -161,7 +162,8 @@ export function passportAuth(config: BlitzPassportConfig): ApiHandler {
                 "/"
 
               if (error) {
-                redirectUrl += "?authError=" + encodeURIComponent(error.toString())
+                redirectUrl +=
+                  "?authError=" + encodeURIComponent(truncateString(error.toString(), 100))
```

Comment:
That looks good, but maybe we can additionally console.error the whole message?

Reply:
done!